### DISCUSSION TITLE
Are the social cards supposed to work on WhatsApp?

### DISCUSSION THREAD

**Discussion by Guts** [squidfunk/mkdocs-material#5248] [Q&A]
A reader of my site told me that it's a shame there are no thumbnails when we share one of our content on WhatsApp. I first replied that there is, if there is a thumbnail, the title and the description.

Then I checked and found out that while social cards (https://squidfunk.github.io/mkdocs-material/setup/setting-up-social-cards/) work great on Twitter/Facebook/LinkedIn, there is nothing displayed in WhatsApp.

![Screenshot_20230322-143050_WhatsApp](https://user-images.githubusercontent.com/1596222/227212707-525c1f90-9232-40c3-8a80-c057fd476482.jpg)

Is this an expected behaviour?

**Comment by squidfunk** [comment]
No it's not expected. Maybe you could investigate and submit a PR? I suspect that some property might be missing for WhatsApp to detect it properly, but I'm not sure. Quick Googling leads to [this StackOverflow question](https://stackoverflow.com/questions/62035459/whatsapp-not-working-with-open-graph-protocol).
